pwndbg/commands/__init__.py

```diff
@@ -218,3 +221,139 @@ def _ArgparsedCommand(*args):
             return function(**vars(args))
         _ArgparsedCommand.__doc__ = self.parser.description
         return Command(_ArgparsedCommand)
+
+## TODO(pick an appropriate filename for these) Common types for argparse.add_argument(type=...), which will also be used for command completion
+
+class ArgparseType(object):
```

Comment:
Probably should have `abc.ABCMeta` metaclass.
http://stackoverflow.com/questions/35673474/using-abc-abcmeta-in-a-way-it-is-compatible-both-with-python-2-7-and-python-3-5